providers/yttcb/clusterbootstrap.yaml

```diff
@@ -87,7 +87,9 @@
 #@ end
 
 #@ def storage_class_configs_exist():
-#@   return data.values.ENABLE_DEFAULT_STORAGE_CLASS and data.values.PROVIDER_TYPE != "tkg-service-vsphere" and data.values.PROVIDER_TYPE != "docker"
+#@   return data.values.ENABLE_DEFAULT_STORAGE_CLASS and (data.values.PROVIDER_TYPE == "vsphere" or
```

Comment:
ENABLE_DEFAULT_STORAGE_CLASS is by default true, which means that this function is going to return true always.